api/optimeet/groups/views.py

```diff
@@ -63,6 +63,25 @@ def add_users_to_group(request, group_id):
     
     return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
 
+@csrf_exempt
+@api_view(['POST'])
+def add_preference_to_group(request, group_id, user_id, format=None):
+        # Check if the user group exists
+    try:
+        group = models.Group.objects.get(group_id=group_id)
```

Comment:
Let's also check here that the `UserGroup` the `group_id` and `user_id` are a part of exist.
```
models.UserGroup.objects.get(group_id=group, user_id=user_id)
```

Reply:
done, but we are not using the user group instance